addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -97,7 +97,7 @@ def get_homepage_data(self):
             'system': platform.system(),
             }
 
-    @http.route('/', type='http', auth='none')
+    @http.route()
```

Comment:
@Julien00859 is this the route you refered to when you said that the override was needed ? :eyes:

Reply:
nah, I was talking about /web and /web/login for their weird auth=none vs auth=public but you don't touch them so I guess it is fine :man_shrugging: